src/open_deep_research/multi_agent.py

```
@@ -1,38 +1,81 @@
-from typing import List, Annotated, TypedDict, operator, Literal
+import os
+import logging
+from typing import List, Dict, Any, Annotated, TypedDict, operator, Literal, Union
```

Ruff (0.11.9)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)